Source/VisualStudioTools/Private/VSTestAdapterCommandlet.cpp

```diff
@@ -25,10 +25,10 @@ static void GetAllTests(TArray<FAutomationTestInfo>& OutTestList)
 static void ReadTestsFromFile(const FString& InFile, TArray<FAutomationTestInfo>& OutTestList)
 {
 	TSet<FString> TestCommands;
+	std::wifstream InStream(*InFile);
 
 	// Wrapping in an inner scope to ensure automatic destruction of InStream object without explicitly calling .close().
```

Comment:
This comment no longer applies since you moved InStream out of the block? Also was there a reason you needed to do that?